worldedit-core/src/main/java/com/fastasyncworldedit/core/history/DiskStorageHistory.java

```diff
@@ -418,6 +419,8 @@ public IntPair readHeader() {
                 final FaweInputStream gis = MainUtil.getCompressedIS(fis);
                 // skip mode
                 gis.skipFully(1);
+                // skip version
+                gis.skipFully(1);
```

Comment:
What happens when reading files without version? Or is that just impossible?

Reply:
It will be impossible. Since we're breaking history compatibility anyway, I figured I'd just add it in now in case it's useful in the future. There'll only be the one version to start with lol